Quickstart script doesn't work

**Describe the bug**
Following the [quickstart](https://github.com/vinyldns/vinyldns/blob/master/README.md#quickstart) instructions from a fresh checkout does not result in a working local VinylDNS:

```
$ ./bin/docker-up-vinyldns.sh 
timeout is set to 60
vinyldns version is set to '0.9.4-SNAPSHOT'
Starting vinyldns and all dependencies in the background...
WARNING: The Docker Engine you're using is running in swarm mode.

Compose does not use swarm mode to deploy services to multiple nodes in a swarm. All containers will be scheduled on the current node.

To deploy your application across the swarm, use `docker stack deploy`.

Creating network "docker_default" with the default driver
Creating vinyldns-ldap       ... done
Creating vinyldns-bind9      ... done
Creating vinyldns-localstack ... done
Creating vinyldns-mysql      ... done
Creating vinyldns-api        ... done
Creating vinyldns-portal     ... done
Waiting for api...
pinging http://localhost:9000 ...
Retrying
...
Retrying
Exceeded retries waiting for http://localhost:9000 to be ready, failing
```

**VinylDNS Version**

`master` branch as of commit 1100f3d1f77ae8e64bf58f1281343ef0fd0ae50d

**To Reproduce**
Steps to reproduce the behavior:
1. `git clone https://github.com/vinyldns/vinyldns.git`
2. `./bin/docker-up-vinyldns.sh`
 
**Expected behavior**
Working local VinylDNS.

**Screenshots**
N/A

**Stack trace or error log output**
If I do a `docker logs vinyldns-api`, it has the following stacktrace:
```
...
Caused by: java.net.UnknownHostException: vinyldns-dynamodb
	at java.net.InetAddress.getAllByName0(InetAddress.java:1281)
	at java.net.InetAddress.getAllByName(InetAddress.java:1193)
	at java.net.InetAddress.getAllByName(InetAddress.java:1127)
	at com.amazonaws.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:27)
	at com.amazonaws.http.DelegatingDnsResolver.resolve(DelegatingDnsResolver.java:38)
	at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:112)
	at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:373)
	at sun.reflect.GeneratedMethodAccessor6.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.amazonaws.http.conn.ClientConnectionManagerFactory$Handler.invoke(ClientConnectionManagerFactory.java:76)
	at com.amazonaws.http.conn.$Proxy4.connect(Unknown Source)
	at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:381)
	at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:237)
	at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:185)
	at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:185)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:83)
	at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:56)
	at com.amazonaws.http.apache.client.impl.SdkHttpClient.execute(SdkHttpClient.java:72)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeOneRequest(AmazonHttpClient.java:1258)
	at com.amazonaws.http.AmazonHttpClient$RequestExecutor.executeHelper(AmazonHttpClient.java:1074)
	... 23 more
```
The issue here seems to be that `docker-compose-build.yml` spins up a local DynamoDB via `localstack`, but `.env` sets `DYNAMODB_ENDPOINT=http://vinyldns-dynamodb:8000`, resulting in the API not being able to find it (the endpoint should probably be `http://vinyldns-localstack:19000`).

**Additional context**
N/A

--- a/docker/.env.build
+++ b/docker/.env.build
@@ -1,0 +1,21 @@
+REST_PORT=9000
+# Do not use quotes around the environment variables.
+MYSQL_ROOT_PASSWORD=pass
+# This is required as mysql is currently locked down to localhost
+MYSQL_ROOT_HOST=%
+# Host URL for queue
+QUEUE_HOST=vinyldns-elasticmq
+
+# portal settings
+PORTAL_PORT=9001
+PLAY_HTTP_SECRET_KEY=change-this-for-prod
+VINYLDNS_BACKEND_URL=http://vinyldns-api:9000
+SQS_ENDPOINT=http://vinyldns-localstack:19007
+DYNAMODB_ENDPOINT=http://vinyldns-localstack:19000
+MYSQL_ENDPOINT=vinyldns-mysql:3306
+USER_TABLE_NAME=users
+USER_CHANGE_TABLE_NAME=userChange
+TEST_LOGIN=true
+
+# api and portal docker tag
+VINYLDNS_VERSION=0.9.4-SNAPSHOT

--- a/docker/docker-compose-build.yml
+++ b/docker/docker-compose-build.yml
@@ -3,7 +3,7 @@ services:
   mysql:
     image: "mysql:5.7"
     env_file:
-      .env
+      .env.build
     container_name: "vinyldns-mysql"
     ports:
       - "19002:3306"
@@ -11,7 +11,7 @@ services:
   bind9:
     image: "vinyldns/bind9:0.0.4"
     env_file:
-      .env
+      .env.build
     container_name: "vinyldns-bind9"
     ports:
       - "19001:53/udp"
@@ -41,7 +41,7 @@ services:
   api:
     image: "vinyldns/api:${VINYLDNS_VERSION}"
     env_file:
-      .env
+      .env.build
     container_name: "vinyldns-api"
     ports:
       - "${REST_PORT}:${REST_PORT}"
@@ -55,7 +55,7 @@ services:
   portal:
     image: "vinyldns/portal:${VINYLDNS_VERSION}"
     env_file:
-      .env
+      .env.build
     ports:
       - "${PORTAL_PORT}:${PORTAL_PORT}"
     container_name: "vinyldns-portal"

